backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs

```diff
@@ -257,16 +268,38 @@ public async Task DeleteMeeting(long meetingId)
             await SendEmailsAsync(meetingId, TemplateType.Cancellation);
             
             var meeting = await _context.Meetings.FirstAsync(meeting => meeting.Id == meetingId);
+            await CancelMeetingsInCalendars(meeting);
             var members = _context.MeetingMembers.Where(member => member.MeetingId == meetingId);
             _context.RemoveRange(members);
             _context.Remove(meeting);
 
             await _context.SaveChangesAsync();
         }
+        
+        private async Task CancelMeetingsInCalendars(Meeting meeting)
+        {
+            var recipients = meeting
+                .MeetingMembers
+                .Select(meetingMember => meetingMember.TeamMember.User.Email);
+
+            foreach (var recipient in recipients)
+            {
+                var user = await _context.Users.FirstOrDefaultAsync(el => el.Email == recipient);
+                
+                if (user is not null)
+                {
+                    var calendar = await _context.Calendars.FirstOrDefaultAsync(el => el.AddEventsFromTeamId == meeting.TeamId && el.UserId == user.Id);
```

Comment:
you take here just first user connected Calendar, but what if he have 3 connected calendars and only last sync with google calendars changes?) I think you need make this changes only in calendars which user sync ( property AddEventsFromTeamId )

Reply:
my trouble, confused with linq query) all is good